var/spack/repos/builtin/packages/gperftools/package.py

```diff
@@ -22,4 +22,18 @@ class Gperftools(AutotoolsPackage):
     version('2.4', sha256='982a37226eb42f40714e26b8076815d5ea677a422fb52ff8bfca3704d9c30a2d')
     version('2.3', sha256='093452ad45d639093c144b4ec732a3417e8ee1f3744f2b0f8d45c996223385ce')
 
-    depends_on("unwind")
+    variant('sized-delete', default=False, description="Build sized delete operator")
+    variant('dynamic-sized-delete-support', default=False, description="Try to build run-time switch for sized delete operator")
+    variant('debugalloc', default=True, description="Build versions of libs with debugalloc")
+    variant('libunwind', default=True, description="Enable libunwind linking")
+
+    depends_on("unwind", when="+libunwind")
+
+    def configure_args(self):
+        args = []
+        args.append(self.enable_or_disable("sized-delete"))
```

Comment:
I think with autotools `enable_or_disable` returns a list, so all these need to be `extends` (or just chain them together using `+`). Did you test this?